### ISSUE TITLE
MailPoet 3 compatibility

### ISSUE THREAD

**Issue by mindctrl** [restrictcontentpro/rcp-mailpoet#7]
We've had a report that this doesn't work with 3.x.

I think version 3 is a total rewrite, though I'm not sure as I haven't looked at their code. They are releasing it under another plugin slug entirely though:
https://wordpress.org/plugins/mailpoet/

the 2.x version is:
https://wordpress.org/plugins/wysija-newsletters/

**Comment by nosegraze** [comment]
Yes it is a total rewrite. I had a brief look at it back when it was in beta, though it's clearly more finalized now and considered "production ready".

**Comment by nosegraze** [comment]
PR: #8

**Comment by nosegraze** [comment]
Got another ticket about this. Can you take a look soon @mindctrl ?

**Comment by mindctrl** [comment]
Doh I missed this somehow. Will review now.

**Comment by mindctrl** [comment]
Works great! Merged.
